### ISSUE TITLE
No Detection Dataset

### ISSUE THREAD

**Issue by psinha30** [zhangzhengfu/EquipmentNameplateDataset#1]
I couldn't find the link for text detection dataset

**Comment by psinha30** [comment]
@zhangzhengfu

**Comment by BothCats** [comment]
Yes, there is no detection dataset for train, can you send me a detection train dataest to me

**Comment by BothCats** [comment]
Yes, there is no detection dataset for train, can you send me a detection train dataest to me

**Comment by BothCats** [comment]
Dear Mr.Zhang:
    Thanks for your reading. Sorry, there is no detection dataset for train. 
    I am a software engineering graduate student at Beijing Key Laboratory of Information Service Engineering which in Beijing Union University. I read your published "The Equipment Nameplate Dataset for Scene Text Detection and Recognition" and two related papers. I appreciate the results your team has achieved, and hope to get your public detection dataset and related algorithms for recurring experiments.
     If you have any suggestion or particulars for the experiment，thast's very grateful. Here‘s my e-mail: ln1950274512@163.com. Looking forward for your reply.

Yours faithfully,
Li Han

**Comment by varshaneya-v** [comment]
@zhangzhengfu, can you please provide the text detection dataset??

**Comment by laofeiwei** [comment]
> I couldn't find the link for text detection dataset

Did you find some text detection dataset?

**Comment by Boo-Booo** [comment]
Who has found this  text detection dataset ? Can you share  with me ?

**Comment by laofeiwei** [comment]
&nbsp;No, the authors did not expose the text location dataset. The article I'm working on will expose the end-to-end labeling dataset of&nbsp; nameplate devices. I'll let you know when it's published if you like.



---Original---
From: ***@***.***&gt;
Date: Thu, Feb 16, 2023 22:12 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [zhangzhengfu/EquipmentNameplateDataset] No Detection Dataset(Issue #1)




 
Who has found this  text detection dataset ? Can you share  with me ?
